odata-csdl/12 Action and Function.md

```diff
@@ -271,6 +271,15 @@ present with the literal value `true`.
 
 Absence of the `$Type` member means the type is `Edm.String`.
 
+### ##subisec Annotation `Core.IsDelta`
+
+The `$ReturnType` of a function or action that returns results as a delta payload
+is annotated with the term [`Core.IsDelta`]($$$OData-VocCore$$$#IsDelta).
```

Comment:
What can the return type be in this case?
* a (collection of an) entity type
* a (collection of a) complex type

Analogous question for parameters

Reply:
Only entity or collection of entity according to Protocol and JSON Format.